mdp/nodes/stats_nodes_online.py

```diff
@@ -0,0 +1,93 @@
+import mdp
+
+
+class MovingAvgNode(mdp.PreserveDimOnlineNode):
```

Comment:
If I understand correctly, this is not really a "moving average" in the usual sense (e.g. https://en.wikipedia.org/wiki/Moving_average). It's more of an online average calculation, so I would suggest to call it "OnlineMeanNode", "OnlineAvgNode", or similar